presto-main/src/main/java/io/prestosql/sql/planner/optimizations/BeginTableWrite.java

```diff
@@ -189,7 +189,8 @@ private WriterTarget createWriterTarget(WriterTarget target)
             }
             if (target instanceof TableWriterNode.RefreshMaterializedViewReference) {
                 TableWriterNode.RefreshMaterializedViewReference refreshMV = (TableWriterNode.RefreshMaterializedViewReference) target;
-                return new TableWriterNode.RefreshMaterializedViewTarget(metadata.beginRefreshMaterializedView(session, refreshMV.getStorageTableHandle()),
+                return new TableWriterNode.RefreshMaterializedViewTarget(
+                    metadata.beginRefreshMaterializedView(session, refreshMV.getStorageTableHandle(), refreshMV.getSourceTableHandles()),
```

Comment:
Nit: the indentation is off here. Continuation indent should be 8 spaces

Reply:
fixed.